browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2428,7 +2428,7 @@ Are you sure you want to do this?
       Try for FREE
     </message>
     <message name="IDS_TRY_7_DAYS_FOR_FREE" desc="Brave VPN plan action text">
-      Try for Free for 7 Days
+      Try 7 Days Free
```

Comment:
This still sounds odd to me, wdyt @rmcfadden3?

Reply:
@mkarolin — yeah, it does read kinda funny. Also, since this is Android, it doesn't have to be in title case. So what about:

`Try 7-day free trial` or `Start 7-day free trial` or `Start free trial` or `Start free VPN trial`

Reply:
i agree. we need to update figma in that case. CC @aguscruiz

Reply:
Ok with me then if this needs to be exactly that string.